src/docs/truffle/getting-started/running-migrations.md

```diff
@@ -113,34 +113,40 @@ Filename: `migrations/1_initial_migration.js`
 ```javascript
 var Migrations = artifacts.require("Migrations");
 
-module.exports = function(deployer) {
-  // Deploy the Migrations contract as our only task
-  deployer.deploy(Migrations);
+module.exports = function(deployer, networks, accounts) {
+  deployer.then(async () => {
+    // Deploy the Migrations contract as our only task
+    await deployer.deploy(Migrations);
+  });
 };
 ```
 
 From here, you can create new migrations with increasing numbered prefixes to deploy other contracts and perform further deployment steps.
 
 ## Deployer
 
-Your migration files will use the deployer to stage deployment tasks. As such, you can write deployment tasks synchronously and they'll be executed in the correct order:
+Your migration files will use the deployer to stage deployment tasks. As such, you can write deployment tasks asynchronously:
 
 ```javascript
 // Stage deploying A before B
 deployer.deploy(A);
 deployer.deploy(B);
 ```
 
-Alternatively, each function on the deployer can be used as a Promise, to queue up deployment tasks that depend on the execution of the previous task:
+Alternatively, use async to deploy queue up deployment tasks that depend on the execution of the previous task:
```

Comment:
I won't really push this too hard but when using JavaScript keywords, variable names, etc. it is conventional in this repo to wrap it in backticks. So if you feel like it, it would be "use `async` to deploy".